.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
```

Comment:
And for this one, can we please also include additional non-template info

Before posting an issue, please confirm:
- [ ] Issue can be reproduced on THIS repository (setup using [quickstart instructions](https://github.com/mdn/express-locallibrary-tutorial#quick-start)).
- [ ] I am using a clean/fresh database
- [ ] I have provided output logs and information about my environment setup below.